seedsource_core/django/seedsource/report_config.py

```diff
@@ -110,15 +110,15 @@ def format_latitude_value(config, is_imperial):
 
 
 def format_latitude_range(values, is_imperial):
-    return '{range:.2f} &deg;N'.format(**values)
+    return '{range:.2f} &deg;'.format(**values)
```

Comment:
Please correct this in the UI also (should be a separate PR, can be branched from `master`).